core/predictor/common/seq_generator.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Very good implementation about paddle model reading and sequence file generation.  I do not recommend to add this file to core/predictor/common/, maybe you can put this file in tools folder so that a user can easily find.

Reply:
thank you. I move it.